### PR TITLE
Fix React Errors

### PR DESCRIPTION
**Pull request by nafiz1001**
- [x] Dashboard
- [ ] Action Dropdown
- Failed to fix the "key" error
- [x] ContainersListContent
- [ ] SamplesListContent
- Could not fix "key" error in the table
- [ ] LibrariesListContent
- I don't have any libraries so i don't know if I did it properly
- [x] IndividualsListContent
- [x] ProcessMeasurementsListContent
- [x] ExperimentRunsListSection
- [x] ProjectsListContent
- [ ] ExperimentRunsListContent
- I don't have any experiment runs so i don't know if I did it properly

### CODE REVIEW

File: frontend/src/components/PaginatedTable.js [hunk 1]
```diff
@@ -18,11 +19,11 @@ const mapStateToProps = state => ({
 
 const actionCreators = {setPageSize};
 
-function PaginatedTable ({
+function PaginatedTable (PaginatedTableProps = {
     columns,
     items,
     itemsByID,
-    rowKey = 'id',
+    rowKey: 'id',
```
Comment (UlysseFG) [inline frontend/src/components/PaginatedTable.js:26]: default values not in the middle...

File: frontend/src/components/containers/ContainersListContent.js [hunk 2]
```diff
@@ -51,7 +52,7 @@ const getTableColumns = (samplesByID, containersByID, containerKinds) => [
             {samples.map((id, i) =>
               <React.Fragment key={id}>
                 <Link to={`/samples/${id}`}>
-                  {withSample(samplesByID, id, sample => sample.name, <span>Loading…</span>)}
+                  {samplesByID[id]?.name ?? "Loading..."}
```
Comment (UlysseFG) [inline frontend/src/components/containers/ContainersListContent.js:55]: nope. withItem() fetches the missing data ...

File: frontend/src/components/containers/ContainersListContent.js [hunk 3]
```diff
@@ -74,7 +75,7 @@ const getTableColumns = (samplesByID, containersByID, containerKinds) => [
       sorter: true,
       render: location => (location &&
         <Link to={`/containers/${location}`}>
-          {withContainer(containersByID, location, container => container.name, "Loading...")}
+          {containersByID[location]?.name ?? "Loading..."}
```
Comment (UlysseFG) [inline frontend/src/components/containers/ContainersListContent.js:78]: same as above...

File: frontend/src/components/containers/ContainersListContent.js [hunk 4]
```diff
@@ -171,18 +196,7 @@ const ContainersListContent = ({
...
-        itemsByID={containersByID}
-        loading={isFetching}
-        totalCount={totalCount}
-        page={page}
-        filters={filters}
-        sortBy={sortBy}
-        onLoad={listTable}
-        onChangeSort={setSortBy}
-      />
+      <PaginatedList {...PaginatedListProps} />
```
Comment (UlysseFG) [inline frontend/src/components/containers/ContainersListContent.js:199]: Is there any reason to refactor these ? Are they raising errors ?

File: frontend/src/components/containers/ContainersListContent.js (hunk 2 above)
Comment (nafiz1001) [reply frontend/src/components/containers/ContainersListContent.js:55]: Not the best place to fetch missing data.

File: frontend/src/components/containers/ContainersListContent.js (hunk 3 above)
Comment (nafiz1001) [reply frontend/src/components/containers/ContainersListContent.js:78]: same as above ;)

**Pull request closed without merging**